gflownet/envs/crystals/spacegroup.py

```diff
@@ -73,7 +88,12 @@ class SpaceGroup(GFlowNetEnv):
     the order of selection of properties.
     """
 
-    def __init__(self, n_atoms: Optional[List[int]] = None, **kwargs):
+    def __init__(
+        self,
+        space_groups_subset: Optional[Iterable] = None,
```

Comment:
There's no docstring for this new attribute

Reply:
https://github.com/alexhernandezgarcia/gflownet/pull/223/commits/4cd042a6981f6c69ac5eacb26ebaa26164e72037